source/OutfitterPanel.cpp

```diff
@@ -112,6 +116,13 @@ void OutfitterPanel::Step()
 		if(!DoHelp("outfitter") && !DoHelp("outfitter 2") && !DoHelp("outfitter 3"))
 			// All help messages have now been displayed.
 			checkedHelp = true;
+
+	for(const auto &outfit : outfitter)
+		if(!player.OutfitIsKnown(*outfit))
+			player.DiscoverOutfit(*outfit);
+
+	for(auto &&ship : player.Ships())
```

Comment:
This one could also be moved to the constructor.

Reply:
will take a look